examples/request-calibrate.json

```diff
@@ -1,21 +1,25 @@
   "params": {
     "exp":   0.16207166221196045,
     "conv":  0.7009195813964052,
     "rec":   0.7040317196117394,
     "death": 0.15807853921067516
-   }
-  
-  
-  
+   },
+  "feature_mappings": {
+    "S": "S",
+    "Exp": "E",
+    "R": "R",
+    "Inf": "I",
+    "D": "D"
+  },
+  "timesteps_column": "timestepz",
```

Comment:
Is this still expecting a list of numbers?
What happens when this value is not valid?

What is this timestepz value?

Reply:
we no longer pass in the data list, we pass a csv in dataset (or an id to a csv on tds). the dataset already has the timesteps and the data but the names might be off so `timesteps_column` is the name of the timesteps column in the dataset and the feature mapping is `name in dataset` -> `name used by model compartments`

Reply:
check out dataset a few lines below. the first column will be called 'timestepz'. hopefully this will demonstrate what i'm talking about

Reply:
ahh that makes total sense thanks!